packages/commons/src/big-query/liquidity-indices-table/pull-data/pullClosestIndices.ts

```diff
@@ -0,0 +1,54 @@
+import { getBigQuery } from '../../client';
+import { getTableFullName } from '../../utils/getTableName';
+import { TableType } from '../../types';
+import { Address } from '../../../utils/convertLowercase';
+
+export type LiquidityIndexEntry = {
+    chainId: number;
+    rateOracle: Address,
+    liquidityIndex: number,
+    blockTimestamp: number
+}
+
+const mapToLiquidityIndices = (row: any) : LiquidityIndexEntry => row;
```

Comment:
this will not work as it is. we need something like this (i.e. we need to apply `bqToNumeric` to BQ BIGNUMBER). Please also follow the pattern and add this to the mappers file associated to the table. 
https://github.com/Voltz-Protocol/v2-off-chain-monorepo/blob/87c68858c55f899f5aa14e41d79a73e2e4e4d858/packages/commons/src/big-query/positions-table/mapper.ts#L4

Reply:
it's weird because when testing this worked. but for safety I followed the pattern & mapped using bqToNumeric [e0c8edf](https://github.com/Voltz-Protocol/v2-off-chain-monorepo/pull/34/commits/e0c8edf872ac0156572fc067bdb129d70fc36e1d)